.editorconfig

```diff
@@ -0,0 +1,27 @@
+; https://editorconfig.org/
+
+root = true
+
+[*]
+insert_final_newline     = true
+charset                  = utf-8
+trim_trailing_whitespace = true
+indent_style             = space
+indent_size              = 2
+
+[*.{yml, yaml}]
+indent_style = space
+indent_size  = 2
+
+[{Makefile,go.mod,go.sum,*.go,.gitmodules}]
+indent_style = tab
```

Comment:
Tab makes sense for files that require it, but I'm against it otherwise. Spaces always render consistently across editors with monospace fonts, even ones that don't support `.editorconfig`. 

For `*.go` files I'd especially prefer them to be space indented.

Reply:
gofmt apparently will turn them back to tabs anyway!

Reply:
TIL, oops.